### PR TITLE
Renames scan lists

### LINKED ISSUE #463: Rename whitelist, blacklist to different terms
Whitelist / blacklist is problematic language.

Maybe
only_scan
never_scan
always_scan

?

### PR DESCRIPTION
**Pull request by TheSkorm**
Renames scan lists to make them make more sense.

Black got a little eager on some things, I can unstage those if you want.

Closes #463

### CODE REVIEW

**Comment by TheSkorm** [comment]
```
Apr 28 19:42:02 luftballon auto_rx[6148]: 2021-04-28 19:42:02,348 WARNING:Config - whitelist configuration has been deprecated and replaced with only_scan list
Apr 28 19:42:02 luftballon auto_rx[6148]: 2021-04-28 19:42:02,349 WARNING:Config - blacklist configuration has been deprecated and replaced with never_scan list
Apr 28 19:42:02 luftballon auto_rx[6148]: 2021-04-28 19:42:02,349 WARNING:Config - greylist configuration has been deprecated and replaced with never_scan list
```
Logs look like this when the old configuration is used

File: auto_rx/autorx/config.py [hunk 1]
```diff
@@ -227,13 +227,51 @@ def read_auto_rx_config(filename, no_sdr_test=False):
...
+        if (
+            config.has_option("search_params", "always_scan")
+            and config.get("search_params", "always_scan") != ""
+        ):  # check if user has new name for scan lists
+            auto_rx_config["always_scan"] = json.loads(
+                config.get("search_params", "always_scan")
+            )
+        else:
+            logging.warning(
+                "Config - greylist configuration has been deprecated and replaced with never_scan list"
```
Comment (darksidelemm) [inline auto_rx/autorx/config.py:270]: This should be always_scan?

Comment (TheSkorm) [reply auto_rx/autorx/config.py]: yes, yes it should

File: auto_rx/auto_rx.py [hunk 2]
```diff
@@ -666,7 +666,7 @@ def main():
         "--frequency",
         type=float,
         default=0.0,
-        help="Sonde Frequency Override (MHz). This overrides the scan whitelist with the supplied frequency.",
+        help="Sonde Frequency Override (MHz). This overrides the scan only_scan list with the supplied frequency.",
```
Comment (darksidelemm) [inline auto_rx/auto_rx.py:669]: scan only_scan list?

**Pull request merged**